target/linux/ath79/dts/ar9342_ubnt_powerbeam-5ac-gen2.dts

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include "ar9342_ubnt_wa.dtsi"
+
+/ {
+	compatible = "ubnt,powerbeam-5ac-gen2", "ubnt,wa", "qca,ar9342";
+	model = "Ubiquiti PowerBeam AC Gen2";
+
+	aliases {
+		led-boot = &led_blue_rssi3;
+		led-failsafe = &led_blue_rssi3;
+		led-upgrade = &led_blue_rssi3;
+		label-mac-device = &pcie;
```

Comment:
That won't work, you need to use 02_network.

Reply:
Done.